docs/eigenai/concepts/eigenai-overview.md

```diff
@@ -17,30 +17,65 @@ To integrate deTERMinal token credit with your client, refer to the [Grant API I
 For more tokens or different models, you can [contact us here](https://ein6l.share.hsforms.com/2L1WUjhJWSLyk72IRfAhqHQ).
 :::
 
-## Overview
-Build verifiable applications leveraging LLM inference without wondering if the same LLM call might produce different results 
+## What is EigenAI? 
+
+EigenAI is a verifiable LLM inference service that provides deterministic execution of open source LLMs.
+Unlike traditional AI services where you trust the provider's outputs, EigenAI enables cryptographic verification that inference
+results are executed using the specified model and input.
+
+EigenAI enables reproducible, and auditable AI inference. Developers 
+access these guarantees through an OpenAI-compatible API and support for frontier open-source models.
+
+:::note
+- OpenAI-compatible refers to the [messages-based Chat Completions API](https://platform.openai.com/docs/api-reference/chat/create).
+- Deterministic refers to providing one request (prompt, parameters, etc) to the EigenAI API multiple times will produce the same output bit-by-bit.
+:::
+
+## Why build with EigenAI? 
+
+Build verifiable applications leveraging LLM inference without wondering if the same LLM request might produce different results
 on different runs, or whether your prompts, models, or responses are modified in any way. EigenAI offers:
 
 * [Deterministic execution of EigenAI API requests.](deterministic-execution.md)
 * [Drop-in compatibility with the OpenAI API.](drop-in-compatibility.md)
 
```

Comment:
Would add a #2 here and mention that verifiable AI provides the rails for developers to instill trust in downstream stakeholders (such as users) that the AI was executed as expected, verifiably so.